test/decl/var/property_wrappers.swift

```diff
@@ -245,7 +245,7 @@ struct Initialization {
   var y = true
 
   @WrapperWithInitialValue<Int>
-  var y2 = true // expected-error{{Bool' is not convertible to 'Int}}
+  var y2 = true // expected-error{{cannot convert value of type 'Bool' to specified type 'Int'}}
```

Comment:
How about `cannot assign value of type 'Foo' to property with 'wrappedValue' of type 'Bar'`?

Reply:
The problem is property can't have `wrappedValue`, in this case property itself is of type `Int` which happened to be inferred from a property wrapped associated with it.

Reply:
Maybe it should say something like `cannot initializer property wrapper 'WrapperWithInitialValue<Int>' with a value of type 'Bool'` but I'm not sure whether that's any better... WDYT, @hborla?

Reply:
Hmm, I guess what you have is fine then. It would've been nice to mention (even as a note) that the property has wrapper(s) attached to it along with the inferred type (like `property 'foo' inferred to have type 'T' via attached property wrapper 'Bar'`), but it's not a big deal.